src/data-fetching/api/PrezlyApi.ts

```diff
@@ -108,8 +107,8 @@ export class PrezlyApi {
     /**
      * Note: this method returns ALL stories from the newsroom. It's intended to be used for sitemaps and not to display actual content.
      */
-    async getAllStories(order: SortOrder = DEFAULT_SORT_ORDER) {
-        const sortOrder = getSortByPublishedDate(order);
+    async getAllStories({ order = DEFAULT_SORT_ORDER, pinning = false }: GetStoriesOptions = {}) {
```

Comment:
This is a breaking change. Will bump major version.

Reply:
I don't see how this is a breaking change 🤔 
`pinning` parameter is optional, and without it, nothing would change, right?

Reply:
It *is* a breaking change. There was a single positional argument of string type. Now it's a destructured object.

Reply:
Oh right, I missed it! Now it makes sense.